src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -40,7 +40,7 @@ def decompress_and_decode_queued_strings(queued_string):
 
 @job('d3a')
 def start(scenario, settings, events):
-    logging.getLogger().setLevel(logging.ERROR)
+    logging.getLogger().setLevel(logging.INFO)
```

Comment:
Please revert to ERROR, because otherwise the d3a staging log will get spammed with info messages, and I am not sure if Kibana can support that many.

Reply:
Still toto